.github/workflows/python-formatting.yml

```diff
@@ -0,0 +1,35 @@
+name: Python Code Format Check
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with: 
+          python-version: 3.8
+
+      - name: Install OS Dependencies
+        run: |
+          sudo apt update
+          sudo apt install python-dev
+          sudo apt purge postgresql-client-14 postgresql-server-dev-all
+          sudo apt install postgresql-12-postgis-3
+
+      - name: Install Poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+
+      - name: Install project
+        run: |
+          poetry install
+
+      - name: Code format check
+        run: poetry run black . --check
```

Comment:
Would it be worth putting this in the `python-ci` workflow? I know that it wouldn't be independent then, but you go through an *awful* lot of duplicated execution to set up the environment.

Reply:
I originally had them in the same workflow but [split them](https://github.com/pacificclimate/scip-backend/pull/1/commits/8fd14d8e236450b039f297f7e0fa1f963d0a4bbf) because the python-ci workflow is pretty slow.  I will switch it back.